src/services/useBoundingBox/index.ts

```diff
@@ -52,8 +53,25 @@ function roundBoundingBox(
   }
 }
 
-export function boundingBoxToString(boundingBox: BoundingBox): string {
-  return `${boundingBox.lonMin}°E ${boundingBox.latMin}°N, ${boundingBox.lonMax}°E ${boundingBox.latMax}°N`
+export function boundingBoxToString(
+  boundingBox: BoundingBox,
+  longitudeStepSize: number,
+  latitudeStepSize: number,
+): string {
+  const asPrecision = (value: number, step: number): string => {
+    const nIntegers = Math.floor(Math.log10(Math.abs(value)))
+    const stepStrs = step !== undefined ? step.toString().split('.') : '1'
+    const nDecimalPlaces = stepStrs.length > 1 ? stepStrs[1].length : 2
+    const nPrecision = Math.round(
+      Math.min(Math.max(nIntegers + nDecimalPlaces, 2), 99),
+    )
+    return value.toPrecision(nPrecision)
```

Comment:
would [toFixed(nDecimalPlaces)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) be the same?